.travis.yml

```diff
@@ -54,6 +54,16 @@ notifications:
     on_failure: always
 jobs:
   include:
+    - stage: test
+      python: 3.7
+      dist: xenial
+      env:
+        - TOXENV=check
+    - stage: test
```

Comment:
This job can be removed. We aren't worried at the moment about the docs. We're really just getting at pytest.